redhat-cop/uncontained.io
# Add Google Analytics

#### What is this PR About?
Add Google Analytics tag to theme; add ID to config.

I also updated the `npm start` command in the readme; there was some old information in there.

#### How should we test or review this PR?
Run the site locally and ensure that the following code block appears in the source:

```
<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
	ga('create', 'UA-107719124-1', 'auto');
	ga('send', 'pageview');
</script>
```
#### Is there a relevant Trello card or Github issue open for this?
No GitHub issue exists, but we discussed this feature in chat.

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----


-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**
